force-app/main/default/classes/SOQL.cls

```diff
@@ -163,9 +217,20 @@ public with sharing class SOQL {
      */
     public static Decimal getTotalRevenueByStage(String stage) {
         // Use AggregateQuery to get the total revenue
+        Decimal totalRevenue = 0;
+
+        AggregateResult[] results = [
+            SELECT SUM (Amount) totalRevenue
+            FROM Opportunity
+            WHERE StageName = :stage OR Amount > 10000
+        ];
+
+        if(results.size() > 0 && results[0].get('totalRevenue') != null){
+            totalRevenue = (Decimal)results[0].get('totalRevenue');
+        }
```

Comment:
Aggregate queries always return at least one result (will be zero if there is nothing to Sum). So these checks are unnecessary